addons/io/org.openhab.io.neeo/src/main/java/org/openhab/io/neeo/NeeoService.java

```diff
@@ -67,11 +66,9 @@
  * @author Tim Roberts - Initial Contribution
  */
 @NonNullByDefault
-@Component(service = org.eclipse.smarthome.core.events.EventSubscriber.class, configurationPolicy = ConfigurationPolicy.OPTIONAL, immediate = true, property = {
+@Component(service = org.eclipse.smarthome.core.events.EventSubscriber.class, immediate = true, property = {
```

Comment:
Just curious - do we need the full package name `org.eclipse.smarthome.core.events.EventSubscriber.class` for the service declaration in this case? Is `EventSubscriber.class` not sufficient?

Reply:
Yes the fully qualified reference was unnecessary. So I've removed it and fixed the merge conflict.